### ISSUE TITLE
WCAG compatibility

### ISSUE THREAD

**Issue by mplusb** [WPChill/modula-lite#608]
**Is your feature request related to a problem? Please describe.**
one technical detail concerning WCAG before we continue our consideration: in the lite version, when a lightbox is activated and then closed, cursor focus returns to the top of the page, not the item that was just closed. this is a problem for keyboard users, who then have to tab through the entire page to access another item in the gallery or items on the page below the gallery. (yes, we know they can use arrows within the lightbox; that's not the scenario we're looking at.)

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
https://secure.helpscout.net/conversation/1645349327/41338?folderId=1289779

**Comment by mplusb** [comment]
Issue Description
Ensures links have discernible text

Element location
a[data-image-id="\31 3"]
Element source

To solve this issue, you need to...
Fix at least one (1) of these issues:
Element does not have text that is visible to screen readers

aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Element has no title attribute

And

Fix the following:
Element is in tab order and does not have accessible text
